src/anaconda/install.sh

```diff
@@ -9,7 +9,7 @@
 
 
 VERSION=${VERSION:-"latest"}
-USERNAME=${USERNAME:-"automatic"}
+USERNAME="${USERNAME:-"${_REMOTE_USER:-"automatic"}"}"
```

Comment:
```suggestion
USERNAME="${USERNAME:-"${_REMOTE_USER:-"_CONTAINER_USER:-"automatic""}"}"
```

Not sure if we should also consider `_CONTAINER_USER` 💭

Reply:
According to the spec, we shouldn't need to. https://github.com/devcontainers/spec/blob/main/proposals/features-user-env-variables.md

>If no remoteUser is configured, _REMOTE_USER is set to the same value as _CONTAINER_USER.